### PR TITLE
ADEN-10723 | UCP-NO-ADS ad-engine platform

### PR DESCRIPTION
**Pull request by Loczi94**

### CODE REVIEW

File: platforms/shared/context/wiki/ucp-wiki-context.setup.ts [hunk 1]
```diff
@@ -1,12 +1,9 @@
-import { context } from '@wikia/ad-engine';
 import { Injectable } from '@wikia/dependency-injection';
 import { WikiContextSetup } from '../../setup/wiki-context.setup';
 
 @Injectable()
 export class UcpWikiContextSetup extends WikiContextSetup {
 	execute(): void {
 		super.execute();
-		// Make sure showAds is set to false in case of any falsy value
-		context.set('wiki.opts.showAds', !!context.get('wiki.opts.showAds'));
```
Comment (fraszczakszymon) [inline platforms/shared/context/wiki/ucp-wiki-context.setup.ts:10]: If showAds is not needed anymore, we don't need whole UcpWikiContextSetup class then

File: platforms/ucp-mobile/setup/context/wiki/ucp-mobile-wiki-context.setup.ts [hunk 2]
```diff
@@ -1,12 +1,9 @@
 import { WikiContextSetup } from '@platforms/shared';
-import { context } from '@wikia/ad-engine';
 import { Injectable } from '@wikia/dependency-injection';
 
 @Injectable()
 export class UcpMobileWikiContextSetup extends WikiContextSetup {
```
Comment (fraszczakszymon) [inline platforms/ucp-mobile/setup/context/wiki/ucp-mobile-wiki-context.setup.ts:5]: Same here

File: platforms/ucp-no-ads/ucp-no-ads-platform.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { ProcessPipeline } from '@wikia/ad-engine';
+import { Injectable } from '@wikia/dependency-injection';
+import { NoAdsMode } from './no-ads-mode';
+
+@Injectable()
+export class UcpNoAdsPlatform {
+	constructor(private pipeline: ProcessPipeline) {}
+
+	execute(): void {
+		this.pipeline.add(NoAdsMode);
```
Comment (fraszczakszymon) [inline platforms/ucp-no-ads/ucp-no-ads-platform.ts:10]: Shouldn't we have `bootstrapAndGetConsent` in the pipeline?

Comment (Loczi94) [reply platforms/ucp-no-ads/ucp-no-ads-platform.ts:10]: Yes, we should. 🙈

**Review by dmnsobczak — APPROVED** [review]

**Comment by dmnsobczak** [comment]
@Loczi94 Please add Label :)

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**